department-of-veterans-affairs/caseflow
# J hoang/appeals 32914

Resolves [32914](https://jira.devops.va.gov/browse/APPEALS-32914?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow+%28old%29&stepId=20)

# Description
Seed data for Correspondence and CorrespondenceDocument

## Acceptance Criteria

## Testing Plan
1. Go to [32914/Test Plan](https://jira.devops.va.gov/browse/APPEALS-32914?workflowName=VA+Caseflow+Sandbox+Story+Enabler+Workflow+%28old%29&stepId=20)

or

1. Go to rails console `rails c`
3. `RequestStore[:current_user] = User.system_user`
4. `Dir[Rails.root.join("db/seeds/*.rb")].sort.each { |f| require f }`
5. `Seeds::Correspondence.new.seed!`
6. Observe that Correspondences are created along with CorrespondenceDocuments


# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
